packages/react-vapor/src/components/iconCard/IconCard.tsx

```diff
@@ -24,6 +24,7 @@ export interface IconCardProps {
     tooltip?: ITooltipProps;
     choices?: IconCardChoice[];
     small?: boolean;
+    singleChoice?: boolean;

```

Comment:
Can I suggest `animateOnHover` instead? Single choice has not much meaning for me